src/constants/oauth.js

```diff
@@ -0,0 +1,5 @@
+const TWITCH_CLIENT_ID = "uv254wnx3p12yjbtf26glmd8xf1nfc";
+const TWITCH_REDIRECT_URI = `${window.location.origin}/%23/oauth/twitch`;
+const TWITCH_SCOPE = "chat%3Aedit+chat%3Aread";
+
+export const TWITCH_AUTH_URL = `https://id.twitch.tv/oauth2/authorize?response_type=token&client_id=${TWITCH_CLIENT_ID}&redirect_uri=${TWITCH_REDIRECT_URI}&scope=${TWITCH_SCOPE}&state=c3ab8aa609ea11e793ae92361f002671"`
```

Comment:
Esse `state` acaba sendo opcional nos casos que não vai ter uma validação, eu recomendaria tirar e adicionar `&force_verify=true` para obrigar a sempre checar qual conta vai fornecer o token, além de permite entrar com outra conta na mesma hora. (E ficou uma aspas dupla sobrando)

```suggestion
export const TWITCH_AUTH_URL = `https://id.twitch.tv/oauth2/authorize?response_type=token&client_id=${TWITCH_CLIENT_ID}&redirect_uri=${TWITCH_REDIRECT_URI}&scope=${TWITCH_SCOPE}&force_verify=true`
```